Gemfile.lock

```diff
@@ -1,45 +0,0 @@
-GEM
```

Comment:
What just came up my mind: This may fail pretty hard when somebody is on a different ruby version? For example when there is one new version of a gem that only works with ruby 2.4.0 and an older version that works with 2.3.3 and older?